src/libertem/analysis/radialfourier.py

```diff
@@ -9,11 +9,32 @@
 from libertem.viz import CMAP_CIRCULAR_DEFAULT, visualize_simple, cmaps
 from .base import AnalysisResult, AnalysisResultSet
 from .masks import BaseMasksAnalysis
-
+from .helper import GeneratorHelper
 
 log = logging.getLogger(__name__)
 
 
+class RadialTemplate(GeneratorHelper):
+
+    short_name = "radial"
+    api = "create_radial_fourier_analysis"
+
+    def __init__(self, params):
+        self.params = params
+
+    def convert_params(self):
+        params = [f'dataset=ds']
+        for k in ['cx', 'cy', 'ri', 'ro', 'n_bins', 'max_order']:
+            params.append(f'{k}={self.params[k]}')
+        return ', '.join(params)
+
+    def get_plot(self):
+        # wrong visualization
```

Comment:
need help for plotting :)

Reply:
![rad_to_upload](https://user-images.githubusercontent.com/59721339/84935303-1fca0c80-b0f6-11ea-9aee-db57f0642cd0.png)

Which one should I choose `ctx.run()` or `ctx.run_udf()`

Reply:
The UDF is just an `ApplyMasksUDF`. The "magic" is in creating a suitable sparse mask stack and interpreting the result. For that reason I'd first go with the Analysis in the notebook and maybe adding a comment to have a look at `RadialFourierAnalysis.get_mask_factories()` and `RadialFourierAnalysis.get_results()` if someone is interested in the inner workings of this analysis.